CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing Guidelines
+This repo contains alpha release of some of the foundational aspects of V2 of Kubernetes Federation. All the contribution is done via GitHub. Please also read the contributors guide: https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
+
+### Sign the CLA
+Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests. Please see https://github.com/kubernetes/community/blob/master/CLA.md for more information
+
+### Learn About Kubernetes Federation v2
+If you want to learn about Kubernetes Federation v2,please read through the doc:
+https://github.com/kubernetes-sigs/federation-v2#kubernetes-federation-v2
+
+To learn more about the Kubernetes Federation v2 project, please read through the User Guide:
+https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/userguide.md
+
+If you are interested in contributing to Kubernetes Federation v2 project, please read through the Development Guide:
+https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/development.md
+
+
+### Submitting a Pull Request (PR)
+1. It is require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests.Please see            https://github.com/kubernetes/community/blob/master/CLA.md for more information
```

Comment:
Just a generic "Sign the CLA" step is all that is needed. Or you can reference the section above in this guide where it talks about signing the CLA.  Let's not duplicate the same content.